plugin/storage/cassandra/factory.go

```diff
@@ -251,3 +252,7 @@ func (f *Factory) Close() error {
 func (f *Factory) PrimarySession() cassandra.Session {
 	return f.primarySession
 }
+
+func (f *Factory) Purge() error {
+	return f.primarySession.Query("TRUNCATE traces").Exec()
```

Comment:
the v1 e2e test should also use this function

Reply:
similar to ES tests, I would recommend first making this change to the existing integration test (i.e. switch it to use Purge and not recreate the factory for every test). Once that works supporting e2e test should be much simpler.

Reply:
#5414